src/screenComponents/combatManeuver.h

```diff
@@ -11,6 +11,12 @@ class GuiCombatManeuver : public GuiElement
 private:
     GuiSnapSlider2D* slider;
     GuiProgressbar* charge_bar;
+	string combat_left_key;
```

Comment:
As these 3 strings are only used during construction, there is no need for them to be members of the class, but can be defined inside the function itself.

Reply:
Yeah, saw that shortly after I submitted the PR. done.